app/liquid_tags/slideshare_tag.rb

```diff
@@ -0,0 +1,33 @@
+class SlideshareTag < LiquidTagBase
+  def initialize(tag_name, key, tokens)
+    super
+    @key    = validate key
```

Comment:
```suggestion
    @key    = validate key.strip
```
Let's pass in a `stripped` key instead, since otherwise it'll fail the regex (extra space makes it 15 total characters).